### PR TITLE
Spanish version of the Eclipse Collections site

### PR DESCRIPTION
**Pull request by HillmerCh**
Signed-off-by: HillmerCh <hilac@msn.com>

### CODE REVIEW

**Comment by donraab** [comment]
Thank you for the great contribution @HillmerCh !  There are some additional changes that will need to be made on each of the individual language pages so they all have a menu link to the Spanish translation.

For Example:

https://github.com/eclipse/eclipse-collections/pull/518/files

We will also need to find someone to review and approve the changes before we merge.

**Comment by nikhilnanivadekar** [comment]
@HillmerCh May be you can have anyone else who reads, writes Spanish comment on the PR if it looks good or suggest changes. Thanks!

**Comment by cchacin** [comment]
@HillmerCh I can review Spanish content, can you please point me to the English version so  I can compare?

/cc @nikhilnanivadekar @donraab FYI

**Comment by HillmerCh** [comment]
Hi @cchacin this is the English version https://github.com/eclipse/eclipse-collections/blob/master/site/index.html

**Comment by nikhilnanivadekar** [comment]
English version: https://github.com/eclipse/eclipse-collections/blob/master/site/index.html

**Comment by HillmerCh** [comment]
> Thank you for the great contribution @HillmerCh ! There are some additional changes that will need to be made on each of the individual language pages so they all have a menu link to the Spanish translation.
> 
> For Example:
> 
> https://github.com/eclipse/eclipse-collections/pull/518/files
> 
> We will also need to find someone to review and approve the changes before we merge.

Hi @donraab, @cchacin is going to review the PR, after that I will add the menu links. 

Thanks

File: site/es/index.html [hunk 1]
```diff
@@ -0,0 +1,665 @@
...
+<head>
+
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+
+    <title>Eclipse Collections - Recursos que quieres usar con colecciones y que necesitas.</title>
```
Comment (cchacin) [inline site/es/index.html]: `Eclipse Collections - Caracteríticas que quieres con las colecciones que necesitas.`

File: site/es/index.html [hunk 2]
```diff
@@ -0,0 +1,665 @@
...
+                        <li><a href="../javadoc/9.0.0">9.0.0</a></li>
+                        <li><a href="../javadoc/8.2.0">8.2.0</a></li>
+                        <li><a href="../javadoc/8.1.0">8.1.0</a></li>
+                        <li><a href="../javadoc/8.0.0">8.0.0</a></li>
+                        <li><a href="../javadoc/7.1.0">7.1.0</a></li>
+                    </ul>
+                </li>
+                <li class="dropdown">
+                    <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
+                       aria-expanded="false">Lenguage<span class="caret"></span></a>
```
Comment (cchacin) [inline site/es/index.html]: `Lenguaje`

File: site/es/index.html [hunk 3]
```diff
@@ -0,0 +1,665 @@
...
+<!-- Header -->
+<a class="anchor" name="about"></a>
+<div class="intro-header">
+    <div class="container-fluid">
+
+        <div class="row">
+            <div class="col-lg-push-2 col-lg-8">
+                <div class="intro-message">
+                    <h1>Eclipse <span style="font-weight: 400">Collections</span></h1>
+                    <h3>- Recursos que quieres usar con colecciones y que necesitas -</h3>
```
Comment (cchacin) [inline site/es/index.html]: `Eclipse Collections - Caracteríticas que quieres con las colecciones que necesitas -`

File: site/es/index.html [hunk 4]
```diff
@@ -0,0 +1,665 @@
...
+<a class="anchor" name="start"></a>
+<div class="content-section-a">
+
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-lg-5 col-sm-6">
+                <hr class="section-heading-spacer">
+                <div class="clearfix"></div>
+                <h2 class="section-heading">Primeros pasos con <br>Eclipse Collections</h2>
+                <p class="lead">Eclipse Collections es el mejor framework de Java para trabajar con colecciones
```
Comment (cchacin) [inline site/es/index.html]: `Eclipse Collections es el mejor framework de colecciones en Java que trae felicidad a tu desarrollo con Java.`

File: site/es/index.html [hunk 5]
```diff
@@ -0,0 +1,665 @@
...
+                    que trae felicidad al desarrollo con Java.
+                    <br><br>Intégralo hoy a su código con sus IDEs y herramientas favoritas!</p>
```
Comment (cchacin) [inline site/es/index.html]: `Intégralo a tu código hoy con tus herramientas de construcción favoritas!`

File: site/es/index.html [hunk 6]
```diff
@@ -0,0 +1,665 @@
...
+
+    <div class="container-fluid">
+
+        <div class="row">
+            <div class="col-lg-push-7 col-lg-5 col-sm-push-6  col-sm-6">
+                <hr class="section-heading-spacer">
+                <div class="clearfix"></div>
+                <h2 class="section-heading">API enriquecidas, concisas y de fácil lectura</h2>
+                <p class="lead">
+                    Maximice el poder de las expresiones Lambda incorporadas en Java desde la versión 8 y los métodos referenciados
```
Comment (cchacin) [inline site/es/index.html]: `Maximiza`

File: site/es/index.html [hunk 7]
```diff
@@ -0,0 +1,665 @@
...
+
+<div class="content-section-a">
+
+    <div class="container-fluid">
+
+        <div class="row">
+            <div class="col-lg-5 col-sm-6">
+                <hr class="section-heading-spacer">
+                <div class="clearfix"></div>
+                <h2 class="section-heading">Varios tipos de contenedores de colecciones que incluyen <br>colecciones inmutables, colecciones primitivas, bimaps, multimaps y sacos/bolsas</h2>
```
Comment (cchacin) [inline site/es/index.html]: sacos/bolsas => `bags`

File: site/es/index.html [hunk 8]
```diff
@@ -0,0 +1,665 @@
...
+                <p class="lead">Puede hacer uso de varios tipos de contenedores de colecciones, aprovechando métodos de factorías para la creación de instancias.</p>
```
Comment (cchacin) [inline site/es/index.html]: métodos de factorías => `métodos de fábrica`

File: site/es/index.html [hunk 9]
```diff
@@ -0,0 +1,665 @@
...
+    <div class="container-fluid">
+
+        <div class="row">
+            <div class="col-lg-5 col-sm-6">
+                <hr class="section-heading-spacer">
+                <div class="clearfix"></div>
+                <h2 class="section-heading">Aprende Eclipse Collections con Katas</h2>
+                <p class="lead">
+                    Revisa <a href="https://github.com/eclipse/eclipse-collections-kata">Eclipse Collections Kata</a>, para aprender el uso idiomático de Eclipse Collections de una manera divertida.
+                    Esta KATA es una configuración de una serie de casos de pruebas con ejercicios por resolver o fallos.
```
Comment (cchacin) [inline site/es/index.html]: `Esta kata está configurada como una serie de ejercicios de pruebas unitarias las cuales fallan.`
`Tu tarea es hacer que pases, usando Eclipse Collections.`

**Comment by HillmerCh** [comment]
I paid attention @cchacin suggestions and Spanish to the Menu links

**Comment by donraab** [comment]
Thank you for the contribution @HillmerCh and the review @cchacin !

**Pull request merged**